app/assets/stylesheets/settings.scss

```diff
@@ -1,4 +1,24 @@
-.ldap-form {
+.setting-form {
   margin: 0 auto;
   max-width: 400px;
 }
+
+.fields {
+  position: relative;
+}
+
+.remove_nested_fields {
+  position: absolute;
+  left: 410px;
+  top: 6px;
+}
+
+select {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.